app-backend/db/src/test/scala/com/azavea/rf/database/PlatformDaoSpec.scala

```diff
@@ -44,7 +44,7 @@ class PlatformDaoSpec extends FunSuite with Matchers with Checkers with DBTestCo
           val dbPlatform = insertPlatformIO.transact(xa).unsafeRunSync
 
           dbPlatform.name == platform.name &&
-            dbPlatform.settings == platform.settings
+            dbPlatform.publicSettings == platform.publicSettings
```

Comment:
You can also check at this stage that `privateSettings == privateSettings` -- since they come back from the database but never get serialized out through the API, we have the information available here to check it I think